### DISCUSSION TITLE
Provide the option to select multiple data source in child queries when mixed data source is selected as parent.

### DISCUSSION THREAD

**Discussion by ravi7248** [grafana/grafana#52377] [Data Sources]
Suppose, I am creating a panel while choosing the mixed data source option.

Now, I can create multiple queries in a single panel while choosing different data sources.
Now, I created one template variable to choose a data source.
Now I want to choose a data source from the template variable in one query.

![image](https://user-images.githubusercontent.com/32006408/179461016-25623ae2-8649-4414-9131-04f43c32f0ab.png)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
